Zend/zend_ast.h

```diff
@@ -74,6 +74,7 @@ enum _zend_ast_kind {
 	ZEND_AST_TYPE,
 	ZEND_AST_CONSTANT_CLASS,
 	ZEND_AST_CALLABLE_CONVERT,
+	ZEND_AST_DEFAULT,
```

Comment:
@iluuu1994 Is this actually correct? I wasn't sure where to put it.

Reply:
Yes, this is correct.